_config/products.yml

```diff
@@ -439,6 +439,34 @@ jbt_core:
             #md5_url: .MD5
 
     luna:
+      4.2.3.CR1:
+        update_site_url: http://download.jboss.org/jbosstools/updates/development/luna/
+        #marketplace_url: http://marketplace.eclipse.org/node/1617241
+        #marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1617241
+        #archived: true
+        release_date: 2015-03-19
+        supported_jbt_is_version: 4.2.0.Final
+        blog_announcement_url: /blog/2015-03-19-JBDS-8.1.0.CR1-for-luna.html
```

Comment:
Should we also fix the Beta1 blog then? https://github.com/nickboldt/jbosstools-website/blob/release423CR1/_config/products.yml#L70